goSrc/src/vmCaller/iroha/commands.go

```diff
@@ -82,6 +369,223 @@ func GetAccountAssets(accountID string) ([]*pb.AccountAsset, error) {
 	}
 }
 
+func GetAccountDetail() (string, error) {
+	query := &pb.Query{Payload: &pb.Query_Payload{
+		Meta: &pb.QueryPayloadMeta{
+			CreatedTime:      uint64(time.Now().UnixNano() / int64(time.Millisecond)),
+			CreatorAccountId: Caller,
+			QueryCounter:     1},
+		Query: &pb.Query_Payload_GetAccountDetail{
+			GetAccountDetail: &pb.GetAccountDetail{}}}}
+	queryResponse, err := makeProtobufQueryAndExecute(IrohaQueryExecutor, query)
+	if err != nil {
+		return "Error", err
+	}
+	switch response := queryResponse.Response.(type) {
+	case *pb.QueryResponse_ErrorResponse:
+		return "ERROR", fmt.Errorf(
+			"ErrorResponse in GetIrohaAccountDetail: %d, %v",
+			response.ErrorResponse.ErrorCode,
+			response.ErrorResponse.Message,
+		)
+	case *pb.QueryResponse_AccountDetailResponse:
+		accountDetailResponse := queryResponse.GetAccountDetailResponse()
+		return accountDetailResponse.Detail, nil
+	default:
+		return "", fmt.Errorf("Wrong response type in GetIrohaAccountDetail")
+	}
+}
+
+func GetAccount(accountID string) (*pb.Account, error) {
+	query := &pb.Query{Payload: &pb.Query_Payload{
+		Meta: &pb.QueryPayloadMeta{
+			CreatedTime:      uint64(time.Now().UnixNano() / int64(time.Millisecond)),
+			CreatorAccountId: Caller,
+			QueryCounter:     1},
+		Query: &pb.Query_Payload_GetAccount{
+			GetAccount: &pb.GetAccount{AccountId: accountID}}}}
+	queryResponse, err := makeProtobufQueryAndExecute(IrohaQueryExecutor, query)
+	if err != nil {
+		return &pb.Account{}, err
+	}
+	switch response := queryResponse.Response.(type) {
+	case *pb.QueryResponse_ErrorResponse:
+		return &pb.Account{}, fmt.Errorf(
+			"ErrorResponse in GetIrohaAccount: %d, %v",
+			response.ErrorResponse.ErrorCode,
+			response.ErrorResponse.Message,
+		)
+	case *pb.QueryResponse_AccountResponse:
+		accountResponse := queryResponse.GetAccountResponse()
+		return accountResponse.Account, nil
+	default:
+		return &pb.Account{}, fmt.Errorf("Wrong response type in GetIrohaAccount")
+	}
+}
+
+func GetSignatories(accountID string) ([]string, error) {
+	query := &pb.Query{Payload: &pb.Query_Payload{
+		Meta: &pb.QueryPayloadMeta{
+			CreatedTime:      uint64(time.Now().UnixNano() / int64(time.Millisecond)),
```

Comment:
This is how it was done in the previous integration.
And it will reduce only one line of code, as again QueryPayloadMeta is part of Query_Payload, so it will have to be in that form.